src/utils/Dictionary.ts

```diff
@@ -132,13 +132,12 @@ export function getCodeFromEvent(event: AllFdEvents, options?: Options): string
 
 export function getCode(suite: string, description: string, events: AllFdEvents[], options?: Options) {
     let firstVisitUrl = '';
-    for (let i = 0; i < events.length; ++i) {
-        const event = events[i];
+    for (const event of events) {
```

Comment:
Wow! I can't believe I wrote that piece of code. Thanks for doing the code-review as well :relieved: